pontoon/base/models.py

```diff
@@ -1272,45 +1272,56 @@ def get_translation(self, plural_form=None):
             }
 
     @classmethod
-    def for_project_locale(self, project, locale, paths=None, filter_type=None,
-        search=None, exclude=None):
+    def for_project_locale(self, project, locale, paths=None, by_type=None,
+        search=None, exclude=None, by_author=None, by_time=None):
         """Get project entities with locale translations."""
-        if filter_type and filter_type != 'all':
-            if filter_type == 'missing':
-                entities = self.objects.missing(locale)
-
-            elif filter_type == 'fuzzy':
-                entities = self.objects.fuzzy(locale)
-
-            elif filter_type == 'suggested':
-                entities = self.objects.suggested(locale)
-
-            elif filter_type == 'translated':
-                entities = self.objects.translated(locale)
-
-            elif filter_type == 'untranslated':
-                entities = self.objects.untranslated(locale)
-
-            elif filter_type == 'has-suggestions':
-                entities = self.objects.has_suggestions(locale)
-
-            elif filter_type == 'unchanged':
-                entities = self.objects.unchanged(locale)
-
-            elif filter_type in Translation.for_locale_project_paths(locale, project, paths).authors().values_list('email', flat=True):
-                entities = self.objects.authored_by(locale, filter_type)
-
-            elif re.match('^[0-9]{12}-[0-9]{12}$', filter_type):
-                try:
-                    start, end = utils.parse_time_interval(filter_type)
-                except ValueError:
+        entities = None
```

Comment:
I think that If you would do the `entities = Entity.objects.all()` here then you would could remove `if entities:` below.